src/main/java/emissary/server/EmissaryServer.java

```diff
@@ -12,7 +13,6 @@
 import emissary.core.Namespace;
 import emissary.core.NamespaceException;
 import emissary.core.ResourceWatcher;
-import emissary.core.sentinel.Sentinel;
```

Comment:
Why the Sentinel removal?

Reply:
100% didn't even know I removed it, completely accidental. Good catch! Added it back in